how-to/add_controller.rst

```diff
@@ -1,119 +1,120 @@
 JAAS: Add controller to JIMM
 ============================
 
-.. warning::
-    This document has not been updated for JIMM.V3.
-
 
 Introduction
 ------------
 
 JIMM gives a centralised view of all models in the system. However the work of managing 
-the models is delegated to a set of standard  juju controllers deployed in various clouds
+the models is delegated to a set of juju controllers deployed in various clouds
 and regions.
 
 These juju controllers must be deployed with some specific options to ensure they work
 correctly in the JAAS system. This document discusses how to bootstrap a juju controller
 such that it will work correctly in a JAAS system.
 
-In this tutorial we will be bootstrapping a new juju controller in AWS and adding it to
-JIMM.
+In this how-to we will show how to add Juju controllers deployed in both MicroK8s and LXD to 
+a JIMM controller.
 
 Prerequisites
 -------------
 
 For this tutorial you will need the following:
 
-- A valid registered domain (regardless of the registrar)
-- AWS credentials
 - Basic knowledge of juju
-- A subdomain registered with Route 53. To learn how to set that up, please follow :doc:`route53`. In this tutorial we will assume that you have registered the canonical.example.com subdomain - please replace this with the appropriate subdomain that you have registered with Route 53.
-- Admin access to a JIMM controller (see this tutorial). For this tutorial we will assume this JIMM is located at ``jimm.canonical.example.com``
+- A JIMM controller deployed in MicroK8s, see :doc:`the tutorial <../tutorial/deploy_jaas_microk8s>`.
+- Administrator permission on the JIMM controller, see ``todo``.
 
-Deploy controller
------------------
 
-1. First we will prepare some parameters for the new controller and export environment variables that we will use in this tutorial. 
+Prelude
+-------
 
-    The **controller name** is the name given to the controller both on the local system and within JIMM. For visibility this often includes the name of the JAAS system, the cloud, the cloud-region and some kind of unique identifier, for example ``jaas-aws-us-east-1-001``. 
+In order for a Juju controller to trust a JIMM controller, a specific config option must be set known
+as the ``login-token-refresh-url``.
 
-    The **cloud** is the cloud in which the controller is being bootstrapped. 
+This config option is set to a specific URL path on the JIMM controller that serves JIMM's public key.
```

Comment:
suggestion: This config option is set to a specific URL path that serves JIMM's public key, which is used to verify signed requests when they reach the Juju controller.